js/server/modules/@arangodb/foxx/router/swagger-context.js

```diff
@@ -470,14 +470,8 @@ module.exports = exports =
         operation.produces.push('application/json');
       }
 
-      for (const param of this._pathParams.entries()) {
-        const name = param[0];
-        const def = param[1];
-        const parameter = (
-        def.schema
-          ? swaggerifyParam(def.schema.isJoi ? def.schema : joi.object(def.schema))
```

Comment:
We now wrap "sloppy" joi object schemas (plain JS objects where every property value is a joi schema) in joi objects during assignment, so there's no longer any need to do this here.